### PR TITLE
Weekly Release 27.04.2016: Enterprise 3.0

### PR DESCRIPTION
**Pull request by bfallonf**
Weekly release

### CODE REVIEW

File: welcome/revhistory_full.adoc [hunk 1]
```diff
@@ -4,6 +4,12 @@
 :data-uri:
 :icons:
 :experimental:
+== Wed Apr 27 2016
+.Install Config
+include::install_config/revhistory_install_config.adoc[tag=install_config_wed_apr_27_2016]
+
+.Architecture
+include::architecture/revhistory_architecture.adoc[tag=architecture_wed_apr_27_2016]
```
Comment (adellape) [inline welcome/revhistory_full.adoc]: @bfallonf Same comment here about fixing the table titles and putting the intro sentence back in place.

**Comment by bfallonf** [comment]
@adellape This good now? Issues addressed and took out the 3.2 commit.

File: architecture/core_concepts/routes.adoc [hunk 2]
```diff
@@ -420,36 +420,95 @@ default certificate] will be used for TLS termination.
...
+<2> The `*termination*` field is `edge` for edge termination.
+<3> The `*certificate*` field is the contents of the PEM format certificate file.
+<4> The `*key*` field is the contents of the PEM format key file.
+<5> An optional CA certificate may be required to establish a certificate chain for validation.
 ====
 
 Because TLS is terminated at the router, connections from the router to
 the endpoints over the internal network are not encrypted.
 
+Edge-terminated routes can specify an `insecureEdgeTerminationPolicy` that
```
Comment (adellape) [inline architecture/core_concepts/routes.adoc]: @bfallonf These lines (454 - 511) are for a 3.1-only feature (see https://github.com/openshift/openshift-docs/pull/1140). It's getting pulled in here from one of the commits. We'll need to remove this before merging to 3.0 docs.

File: welcome/revhistory_full.adoc (hunk 1 above)
Comment (adellape) [inline welcome/revhistory_full.adoc]: @bfallonf These issues don't look fixed?

**Comment by adellape** [comment]
@bfallonf There were a couple problems I was going to point out in this 3.0 one related to the https://github.com/openshift/openshift-docs/pull/1889 PR, but in looking at it more closely I don't think we should be including it, at least not as is (e.g., it's updated to the 3.1+ style /etc/origin/ file names, it talks about the split master api/controllers which is 3.1+ only). I talked with @ahardin-rh and we removed the enterprise-3.0 label.

**Comment by bfallonf** [comment]
@adellape Done. Let me know if there's anything else.

**Comment by adellape** [comment]
@bfallonf Sorry I might not have been clear in my last comment. The commit from https://github.com/openshift/openshift-docs/pull/1889 should be taken out of this OSE 3.0 PR.

**Comment by bfallonf** [comment]
@adellape @vikram-redhat Ok. Removed bad commit and fixed rev histories. Let me know if this fixes it all or merge away.

**Comment by adellape** [comment]
@bfallonf LGTM! Merging.

**Pull request merged**